style.css

```diff
@@ -0,0 +1,627 @@
+  color: #fff;
+  font-family: 'Lato', 'Courier New', Courier, monospace;
+  font-weight: 500;
+  text-align: center;
+  padding-top: 30px;
+  font-size: 24px;
+}
+
+.span-red {
+  height: 2px;
+  width: 45px;
+  background-color: #ec5242;
+  display: block;
+  margin: 15px 0 25px 0;
+  align-self: center;
+}
+
+.day-one,
+.day-two,
+.day-three,
+.day-after,
+.day-previous {
+  color: #fff;
+  display: flex;
+  margin: 0 15px 10px 15px;
+  justify-content: space-between;
+  align-items: center;
+  background-color: rgb(253 252 252 / 40%);
+  padding: 8px;
+}
+
+.cal-img {
+  width: 51px;
+}
+
+.program-subtitle {
+  color: #ff4f38;
+  font-family: 'Lato', 'Courier New', Courier, monospace;
+  font-weight: 500;
+  font-size: 21px;
+}
+
+.program-text {
+  font-family: 'Lato', 'Courier New', Courier, monospace;
+  font-size: 17px;
+  width: 160px;
+}
+
+.buy-tickets {
+  align-self: center;
+  background-color: #ec5242;
+  width: 265px;
+  height: 60px;
+  margin: 25px 40px 45px 40px;
+  color: #fff;
+  text-decoration: none;
+  font-family: 'Lato', 'Courier New', Courier, monospace;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-size: 18px;
+  font-weight: 500;
+}
+
+.featured-artists {
+  display: flex;
+  flex-direction: column;
+}
+
+.artist-title {
+  color: #4d4d4d;
+  font-family: 'Lato', 'Courier New', Courier, monospace;
+  font-weight: 500;
+  text-align: center;
+  padding-top: 30px;
+  font-size: 24px;
+}
+
+.artist-img {
+  width: auto;
+  height: 100px;
+}
+
+.artist-box {
+  display: flex;
+  flex-direction: row;
+  margin: 40px 0 40px 40px;
+}
+
+.artist-info-box {
+  margin: 0 5px 0 10px;
+}
+
+.artist-subtitle {
+  color: #4d4d4d;
+  font-family: 'Lato', 'Courier New', Courier, monospace;
+  font-size: 22px;
+  margin-bottom: 8px;
+}
+
+.artist-intro {
+  color: #ff4f38;
+  font-family: 'Lato', 'Courier New', Courier, monospace;
+  font-size: 15px;
+  margin-bottom: 8px;
+  font-weight: 500;
+  font-style: italic;
+}
+
+.artist-info {
+  color: #4d4d4d;
+  font-family: 'Lato', 'Courier New', Courier, monospace;
+  font-size: 13px;
+}
+
+.future,
+.lil-uzi,
+.kodak-black,
+.lil-wayne {
+  display: none;
+}
+
+.more-btn,
+.less-btn {
+  margin-bottom: 55px;
+  width: 90%;
+  align-self: center;
+  border: 1px solid #4d4d4d;
+  color: #4d4d4d;
+  height: 45px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-family: 'Lato', 'Courier New', Courier, monospace;
+  font-size: 15px;
+  text-decoration: none;
+}
+
+.expand-icon {
+  height: 17px;
+  align-self: center;
+  justify-self: center;
+  margin-left: 5px;
+}
+
+.upper-bar {
+  display: none;
+}
+
+.logo-navbar {
+  display: none;
+}
+
+.sections-navbar {
+  display: none;
+  flex-direction: column;
+  align-items: center;
+  width: 100%;
+  position: absolute;
+  background-image: url(./imgs-capstone/background-one.jpg);
+  background-repeat: no-repeat;
+  background-size: auto 571px;
+  background-position-x: -332px;
+}
```

Comment:
- [ ] Please kindly fix the menu design, the position should be `fixed` then the menu should cover the page... It should be almost the same as in the figma design but the color or background, the position of the closing menu can change. Take a look 👇 👇 

Your design

> |     | Home page                      | About page                      |     |
> | --- | ----------------------------------- | ------------------------------------ | --- |
> |     | <img width="326" alt="Screenshot 2023-04-19 at 22 00 48" src="https://user-images.githubusercontent.com/74501523/233174353-7b606535-226d-4b79-b7e6-e14f7f63577e.png"> | <img width="386" alt="Screenshot 2023-04-19 at 21 47 01" src="https://user-images.githubusercontent.com/74501523/233173198-4851df29-2bb0-4636-adaf-463268e7ea56.png"> |

An example of how the menu should look

<img width="386" alt="Screenshot 2023-04-19 at 21 54 19" src="https://user-images.githubusercontent.com/74501523/233173294-ca1f97d8-1b99-42d3-94b7-08fc4fd1df36.png">